root/programs/C#/Samples/WebApp_sample/ProjectX_sample/Aspx/Framework/DialogFrame.htm

```diff
@@ -3,8 +3,6 @@
 <html>
 	<head>
 		<title>ダイアログ画面</title>
-	</head>
-	<frameset cols="100%" style="margin:0px; padding:0px; border-style:none; background-color:#cccccc;">
-        <frame id="frame1" src="./DialogLoader.htm" style="background-color:#cccccc; " />
-    </frameset>
+	</head>	
+    <iframe id="frame1" src="./DialogLoader.htm" frameborder="0" style="overflow:hidden;overflow-x:hidden;overflow-y:hidden;height:100%;width:100%;position:absolute;top:0px;left:0px;right:0px;bottom:0px;background-color:#cccccc;"/>
```

Comment:
@yusukemaegawa 
I was fixed this pointed out.
https://github.com/SymphonyTeleca/OpenTouryo/blob/nishino/root/programs/C%23/Samples/WebApp_sample/ProjectX_sample/Aspx/Framework/DialogFrame.htm

Please merge.

@kishoreandukuri 
Please tell me the reason that the check of this modifitication was insufficient.